docs/source/evaluation_scripts.md

```diff
@@ -80,3 +82,67 @@ Let's break down what is happening in the above code snippet.
 1. `output` should contain a key named `result`, which is a list containing entries per dataset split that is available for the challenge phase in consideration (in the function definition of `evaluate()` shown above, the argument: `phase_codename` will receive the _codename_ for the challenge phase against which the submission was made).
 2. Each entry in the list should be a dict that has a key with the corresponding dataset split codename (`train_split` and `test_split` for this example).
 3. Each of these dataset split dict contains various keys (`Metric1`, `Metric2`, `Metric3`, `Total` in this example), which are then displayed as columns in the leaderboard.
+
+### Writing a remote evaluation script
+
+Just like a regular evaluation script, a remote evaluation script allows you to customize evaluation for your remote challenge. The starter template for remote challenge evaluation can be found [here](https://github.com/Cloud-CV/EvalAI-Starters/blob/master/remote_challenge_evaluation/evaluation_script_starter.py).
+
+There are a few things that you need to configure:
+
+1. **Set variables in the `__init__` method**: There are few variables that you need to set according to your challenge. The `auth_token` is fetched from the profile and `evalai_api_server` is set according to the server. The `challenge_pk` and `queue_name` can be fetched from the `Manage` tab of your challenge once it is up on  the server.
```

Comment:
Replace this with  

```
Setup Configs: To configure authentication for the challenge set the following environment variables

1.  AUTH_TOKEN: To fetch the auth token go your [profile page](add link) on EvalAI.
2. API_SERVER: Use `https://eval.ai` when setting up challenge on production server. Otherwise, use `https://staging.eval.ai`
3. QUEUE_NAME: Go to the challenge manage tab to fetch the challenge queue name
4. CHALLENGE_PK: Go to the challenge manage tab to fetch the challenge pk
5. SAVE_DIR: Path to submission data download location
```

Reply:
Added the steps from the host challenge page to here as a first step. These are already included there. I have changed this one accordingly.